reframe/utility/sanity.py

```diff
@@ -238,6 +238,7 @@ def contains(seq, key):
 def assert_true(x, msg=None):
     '''Assert that ``x`` is evaluated to ``True``.
 
+    :arg msg: The error message to use if the assertion fails.
```

Comment:
We should mention that somebody can use placeholders in the message. I can fix that and push.

Reply:
Ok, this is documented at the beginning of the section, globally for all the functions. See here: http://localhost:8000/sanity_functions_reference.html#categories-of-sanity-functions

That's why I haven't put anything about the message in each individual function. It was added later and I was lazy :-)